### ISSUE TITLE
Issues with enforced ASCII decode when not ascii

### ISSUE THREAD

**Issue by Robbie1977** [mhe/pynrrd#23]
my files are generally encoded in utf8 and hence I am now getting:

UnicodeDecodeError: 'ascii' codec can't decode byte 0xb5 in position 14: ordinal not in range(128)

```
    341     for raw_line in it:
    342         headerSize += len(raw_line)
--> 343         raw_line = raw_line.decode('ascii')
    344
    345         # Trailing whitespace ignored per the NRRD spec
```

**Comment by Robbie1977** [comment]
It's tricky before loading the header to know what to use so I'd suggest using sys.getdefaultencoding() as a way to let this be modified.

**Comment by Robbie1977** [comment]
Adding .decode('ascii', 'ignore') would remove any unrecognised characters

**Comment by mhe** [comment]
Fixed with merged PR #24.
